src/services/tourist-service.js

```diff
@@ -22,8 +22,22 @@ class TouristService {
         return await request.post('log-out', { token })
     }
 
-    register(name, email, password) {
-        // return
+    async register({ nickName: nickname, ...other }) {
+          try {
+              const res = await request.post('register', {nickname, ...other});
+              localStorage.setItem('token', res.accessToken);
+              localStorage.setItem('refreshToken', res.refreshToken);
+          } catch (e) {
+              
```

Comment:
Handle this error

Reply:
I handle it in register component      
const submit = async (values) => {
        try{
            await api.register(values);
            onAuth();
            setTimeout(() => setRedirect(true), 1000);
        } catch (e) {
            setErr(e);
        }
    };